ext/GeometryOpsDimensionalDataExt/GeometryOpsDimensionalDataExt.jl

```diff
@@ -0,0 +1,21 @@
+module GeometryOpsDimensionalDataExt
+
+import DimensionalData as DD
+import GeometryOps as GO
+import GeoInterface as GI
+
+function GO.polygonize(A::DD.AbstractDimArray; dims=(DD.X(), DD.Y()), crs=GI.crs(A), kw...)
+    lookups = DD.lookup(A, dims)
+    bounds_vecs = if DD.isintervals(lookups)
+        map(DD.intervalbounds, lookups)
+    else
+        @warn "`polygonsize` is not possible for `Points` sampling, as polygons cover space by definition. Treating as `Intervals`, but this may not be appropriate"
```

Comment:
Doesn't this just mean that the ring vertices are located at the points?  That seems pretty exact...

Reply:
Hmm not sure I understand you. It's just saying that points don't cover any space so polygonizing them doesn't make sense

Reply:
I was thinking of it more in a sense that the polygonizing operation is the concave hull of the set of points that satisfy `f`...

Reply:
Yeah, somehing else! `polygonize` is like a groupby/combine for lots of little square polygons